### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by lili2311**
## Linked Jira/Github Issue Ticket

[WHFNP-61](https://womenhackfornonprofits.atlassian.net/browse/WHFNP-61)
## What are the changes included in this PR?
- Changed the non-profits image
- removed commented out code
## If the changes are visual, provide a screenshot.

screen shot 2016-02-20 at 15 35 43

### CODE REVIEW

**Comment by lili2311** [comment]
@tanyapowell  please review

**Comment by EChesters** [comment]
On the image, is that woman representing a non-profit? I can't tell the difference, she looks like a volunteer.

**Comment by Nandhini31** [comment]
Yes she is, we have approval only from her as a non profit to publish on the site and Gandhi photo and quote i thought reflects "cause" : we actually don't have  better pic

File: src/css/screen.css [hunk 1]
```diff
@@ -953,7 +956,6 @@ table {
   .project-img img {
     height: 26em;
     border: 2em solid white;
-    /* border-bottom-width: 0; */
```
Comment (tanyapowell) [inline src/css/screen.css]: could we delete all commented code?

**Comment by tanyapowell** [comment]
:+1: all good with me

**Pull request merged**